analysis/src/Packages.ml

```diff
@@ -13,7 +13,7 @@ let makePathsForModule ~projectFilesAndPaths ~dependenciesFilesAndPaths =
 
 let getReScriptVersion () =
   (* TODO: Include patch stuff when needed *)
-  let defaultVersion = (10, 1) in
+  let defaultVersion = (11, 0) in
```

Comment:
This is intentional. We can easily roll back to defaulting to 10.1 if this causes troubles (it shouldn't).